infrastructure/cdn-in-a-box/traffic_ops/config.sh

```diff
@@ -129,7 +129,8 @@ cat <<-EOF >/opt/traffic_ops/app/conf/cdn.conf
         "password" : "",
         "address" : ""
     },
-    "influxdb_conf_location": "/opt/traffic_ops/app/conf/production/influx.conf"
+    "InfluxEnabled": true,
```

Comment:
good catch - I do wish the config parameter followed the `snake_case` style but not a change for this PR!

Reply:
lets clean it up in another PR  and clean it up across the board - this PR looks good